src/ragas/metrics/_domain_specific_rubrics.py

```diff
@@ -142,6 +142,12 @@ async def _ascore(self, row: t.Dict, callbacks: Callbacks) -> float:
             reference=reference,
             reference_contexts=reference_contexts,
         )
+
+        rubrics_text = "\n".join(
```

Comment:
shouldn't this be part of init? this setup is applied when initializing the rubric right?